src/Evolution/Systems/Cce/Tags.hpp

```diff
@@ -0,0 +1,232 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/SpinWeighted.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+namespace Cce {
+
+/// Tags for Cauchy Characteristic Extraction routines
+namespace Tags {
+
+// Bondi parameter tags
+
+/// Bondi parameter \f$\beta\f$
+struct Beta : db::SimpleTag {
```

Comment:
For consistency with the other tags, this should probably also be `BondiBeta` (else I'd wonder what makes it special)